core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    @content;
+  }
+}
```

Comment:
Also should we mention the mixins will accept the units `dppx` and `dpcm`? And do we want to cater for converting those when a "dpr" is used? I'm not even sure what the conversion is? Also I’m assuming Autoprefixer will handle browsers that don’t support these units e.g. IE 9/10/11, [see](http://caniuse.com/#feat=css-media-resolution).